pgkit/application/models/replica.py

```diff
@@ -257,7 +263,8 @@ def _get_current_delay(self):
         config = read_file(file_location)
         for line in config.split('\n'):
             if 'recovery_min_apply_delay' in line:
-                return int(line.split(' ')[2].split("min")[0]) / 60
+                time = line.split(' ')[2].split("min")[0]
+                return int(0 if time=="''" else int(time[0])) / 60
```

Comment:
Maybe `int(time)`

Reply:
`return int(time or 0) / 60`

Reply:
Please do not change args, remove `... / 60`. that can be dangerous.

Reply:
when I add the following logs to the code 

`
            if 'recovery_min_apply_delay' in line:
                time = line.split(' ')[2].split("min")[0]
                print("time: "+time)
                print("time or 0: "+time or "0")
                print("int(time or 0): "+ str(int(time or 0)))
                return int(0 if time=="''" else int(time)) / 60

`
I get the following result. 

`
time: ''
time or 0: ''
ValueError: invalid literal for int() with base 10: "''"
`

this is why I used the following if.

Reply:
what is your python version?

Reply:
Python 3.8.10

Reply:
I tested it with Python 3.9.5 and it had the same error.

Reply:
As I said please try python 3.9.12 or higher

Reply:
I used Python 3.10.10 and got the same result

Reply:
That's odd. Because I get the following result on 3.9.12
`Python 3.9.12 (tags/v3.9.12:b28265d, Mar 23 2022, 23:52:46) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> time = ''
>>> time or 0
0`

Reply:
What if we check whether the line is starting with `recovery_min_apply_delay`.
In that case, if `recovery_min_apply_delay` is commented parsing the line won't happen.
@SadeghHayeri What's your opinion?

Reply:
We now use `line.startswith` in the if condition.